Templates/TodoList-NodeJS/readme.md

```diff
@@ -0,0 +1,14 @@
+## Todo List with Node.js via HttpTrigger & Storage Table
+HttpTrigger allows you to implement CRUD implementation over Rest API. This template shows how to use a httptrigger create and read entities that resembles todo list entities from Storage Table.
```

Comment:
fix up wording

"This template implements a simple HTTP CRUD API over an Azure Storage Table, allowing todo item entities to be created and read."